tools/sense_studio/templates/frame_annotation.html

```diff
@@ -8,6 +8,16 @@
 
 {% block main %}
 <div id="main" class="ui main container">
+    <div class="ui segment right floated">
+        <h3 class="ui header center aligned">Settings</h3>
+        <div class="ui toggle checkbox hashoverpopup"
+           data-content="Train a simple logistic regression model live on temporal annotations and show current predictions?"
+           data-position="left center">
+            <input type="checkbox" id="logregInput" {% if project_config.show_logreg %} checked="checked" {% endif %}
+               onclick="toggleShowPredictions('{{ path }}', '{{ split }}', '{{ label }}');">
+            <label>Show Predictions</label>
```

Comment:
Guillaume recently called this "Assisted tagging" and I really like that term, because it highlights the purpose of this feature more. Can we use that name throughout the code?